build.gradle

```diff
@@ -155,6 +156,7 @@ subprojects {
             google_api_protos: 'com.google.api.grpc:proto-google-common-protos:2.0.1',
             google_auth_credentials: "com.google.auth:google-auth-library-credentials:${googleauthVersion}",
             google_auth_oauth2_http: "com.google.auth:google-auth-library-oauth2-http:${googleauthVersion}",
+            google_http_client_jackson2: "com.google.http-client:google-http-client-jackson2:${googlehttpVersion}",
```

Comment:
Why google_http_client_jackson2?

It's because grpc-xds uses a class from the artifact.

```
> Task :grpc-xds:compileJava FAILED
/Users/suztomo/grpc-java/xds/src/main/java/io/grpc/xds/internal/sts/StsCredentials.java:29: error: package com.google.api.client.json.jackson2 does not exist
import com.google.api.client.json.jackson2.JacksonFactory;
```